lib/mergePack.js

```diff
@@ -0,0 +1,24 @@
+import merge from 'lodash.merge';
+import fs from 'fs';
+import path from 'path';
+
+export const getPackage = () => {
+  let origPackage;
+  try {
+    origPackage = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'package.json'), 'utf-8'));
+  } catch (e) {
+    origPackage = false;
+  }
+  return origPackage;
+};
+
+export const mergePackages = (origPackage, destinations) => compiledFiles => {
+  if (origPackage) {
+    const newPackagePath = path.join(process.cwd(), 'package.json');
+    const packageIndex = destinations.indexOf(newPackagePath);
+    const packageContents = JSON.parse(compiledFiles[packageIndex]);
+    const mergedPack = JSON.stringify(merge(origPackage, packageContents), null, 2);
```

Comment:
I am not quite sure but I reckon it should be the other way around. package from template should not override fields in already existing package. Only add missing ones.

I mean if I used this template and later changed how my lib is tested, I'd prefer to keep it as it is. But if there are new scripts added, it would be nice to add them to my lib too.

@ericelliott  what is your idea on merging package.json?